tests/resources/iva-defects.json

```diff
@@ -1659,11 +1659,11 @@
       },
       {
         "rsNumber": "1",
-        "requiredStandard": "A vehicle MUST be fitted with either a mechanical anti – theft  device or an electronic immobiliser",
+        "requiredStandard": "A vehicle MUST be fitted with either a mechanical anti – theft device or an electronic immobiliser",
```

Comment:
Are these missing full stops at the end of the RS?

Reply:
![image](https://github.com/dvsa/cvs-svc-defects/assets/79578557/f2c64e51-cc4d-4ea9-b349-c29b3c490abf)